src/Illuminate/Mail/Mailer.php

```diff
@@ -383,7 +383,7 @@ protected function parseView($view)
      * Send a Swift Message instance.
      *
      * @param  \Swift_Message  $message
-     * @return void
+     * @return int|null
```

Comment:
yeh, exactly - that's us saying you probably shouldn't use it, but if you really want, you can